pkgs/data/fonts/vollkorn/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation rec {
+  pname = "vollkorn";
+  version = "4.105";
+
+  src = fetchzip {
+    url = "http://vollkorn-typeface.com/download/vollkorn-4-105.zip";
+    sha256 = "1f4bvaj38gs7msihn6vpvnd0zhnkv966l3sh63dsn2dh00dgsvm0";
+    stripRoot = false;
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
```

Comment:
```suggestion
```

overwriting phases is not allowed.

Reply:
"not allowed" in a general case, did I miss some rules in the manual or CONTRIBUTING.md about it? Because I thought overwriting phases to be a common thing.
Is it preferred to disable standard phases via their boolen flags like `dontBuild = true;`?